CHANGES.rst

```diff
@@ -1,4 +1,7 @@
-0.6.1 (unreleasd)
+0.6.2 (unreleased)
+================
```

Comment:
parser might barf because the line of "=" characters is not as long as the text above it.